.gitignore

```diff
@@ -1,2 +1,8 @@
 .bundle
 Gemfile.lock
+
+# generated rubocop file
+.rubocop-https*
```

Comment:
We ignore `.rubocop-*` in the plugin generator

Reply:
What it means? I added it, because `.rubocop-https---raw-githubusercontent-com-ManageIQ-guides-master--rubocop-base-yml` this file is always autogenerated locally

Reply:
btw core, persister, api & collectors doesn't ignore `.rubocop-*` (which includes `.rubocop-cc.yml` and `.rubocop_local.yml`

Reply:
ah, sorry, I got it, these files are empty now, removing

Reply:
> What it means? I added it, because .rubocop-https---raw-githubusercontent-com-ManageIQ-guides-master--rubocop-base-yml this file is always autogenerated locally

See:
https://github.com/ManageIQ/manageiq/blob/master/lib/generators/manageiq/plugin/templates/.gitignore#L1
https://github.com/ManageIQ/manageiq/blob/master/lib/generators/manageiq/plugin/templates/.rubocop.yml
https://github.com/ManageIQ/manageiq/blob/master/lib/generators/manageiq/plugin/templates/.rubocop_cc.yml
https://github.com/ManageIQ/manageiq/blob/master/lib/generators/manageiq/plugin/templates/.rubocop_local.yml

> btw core, persister, api & collectors doesn't ignore .rubocop-* (which includes .rubocop-cc.yml and .rubocop_local.yml

I think @Fryguy forgot to run some repo setup scripts.  BTW, `.rubocop-*` does not match `.rubocop_cc.yml` or `.rubocop_local.yml` and those files should exist even if they are blank.

Reply:
ok, it should be ok now